CONTRIBUTING.md

```diff
@@ -0,0 +1,151 @@
+# Phootwork Contributing Guide
+
+If you're reading this document, it is likely that you have decided to contribute to this project.
+Therefore, first of all, we want to warmly thank you! The Open Source Software grows and improves
+thanks to people like you!
+
+Contributing to a project is a process with its own rules and we try to explain them along this document.
+
+## Quick Start For Experienced Programmers
+
+1. Fork, clone and apply your patches. See the [directory structure](#understanding-the directory-structure) explanation
+if needed and don't forget to write tests.
+2. Run the test suite `composer test` and fix all red tests.
+3. Run static analysis tool (by now, we use [Psalm](https://psalm.dev/)) `composer analytics` and fix all errors and issues.
+4. Fix the coding standard `composer cs-fix`.
+
+Phootwork also provides a command to run all the tests and analytics, required for a pull request:
+```
+composer pr
+```
+
+## Requirements
+
+To contribute to this project, you should have an good knowledge of PHP (of course) and a basic knowledge of
+[git](https://git-scm.com/), [Github](https://github.com/) and the dependency manager [composer](https://getcomposer.org/). 
+
+## First Step: Clone The Project And Install The Dependencies
```

Comment:
Almost... I found this: https://capitalizemytitle.com/ 😂

Reply:
Fixed each title thanks to that site. I don't understand the rationale under those strange capitalizations and I accept them with religious faith.